src/controllers/autoCompletion.controller.js

```diff
@@ -2,46 +2,50 @@ const jwt = require("jsonwebtoken");
 const Query = require("../models/Query");
 const User = require("../models/User");
 
-const MAXIMUM_AUTO_COMPLETIONS = 10;
-
 exports.getAutoCompletions = async function (req, res, next) {
+  const { accessToken } = req.cookies;
   const { userInput } = req.query;
+  const MAXIMUM_AUTO_COMPLETIONS = 5;
+  const searchHistories = [];
 
-  if (!userInput) {
-    res.status(200).send({
-      result: "ok",
-      searchHistories: [],
-    });
+  let userData;
+  let user;
 
-    return;
+  if (accessToken) {
+    userData = jwt.verify(accessToken, process.env.JWT_SECRET_KEY);
+    user = userData.userId;
   }
 
-  const searchHistories = [];
-
-  if (req.user) {
-    const user = await User.findById(req.user).lean();
-    const userSearchHistory = user.searchHistory;
+  if (user) {
+    const foundUser = await User.findOne({ _id: user }).lean();
```

Comment:
이부분 findById로 바꿀 수 있을 것 같습니다!
```suggestion
    const foundUser = await User.findById(user).lean();
```

Reply:
그렇군요! 수정하겠습니다!